ext/data/recommended-dictionaries.json

```
@@ -26,5 +26,551 @@
         ],
         "grammar": [],
         "pronunciation": []
+    },
+    "zh": {
+        "terms": [
+            {
+                "name": "kty-zh-en",
+                "downloadUrl": "https://github.com/themoeway/kaikki-to-yomitan/releases/latest/download/kty-zh-en.zip",
+                "description": "Chinese to English dictionary, provided by kaikki",
```

```suggestion
                "description": "Chinese to English dictionary created from Wiktionary data.",
```

Maybe this is clearer? I don't think most people know or care about kaikki